### PR TITLE
[Concept Entry] Rust Conditionals

### PR DESCRIPTION
**Pull request by Captainnik53**
added concept entry for conditionals in Rust

### Description

Added concept of Conditional statements in Rust documentation along with appropriate syntax and code snippets.

### Type of Change

- Adding a new entry

### Checklist

- [✅ ] All writings are my own.
- [✅  ] My entry follows the Codecademy Docs style guide.
- [ ✅ ] My changes generate no new warnings.
- [ ✅ ] I have performed a self-review of my own writing and code.
- [ ✅ ] I have checked my entry and corrected any misspellings.
- [✅  ] I have made corresponding changes to the documentation if needed.
- [✅  ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [✅  ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ✅ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/rust/concepts/conditionals/conditionals.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Conditionals'
+  - 'Control Flow'
+CatalogContent:
+  - 'paths/computer-science'
+  - 'learn/rust-for-programmers'
+---
+
+**Conditional** statements in Rust allow the control of flow in program based on certain conditions. Rust provides support for Conditional statements that include `if`, `else if` and `else` statements. Unlike the rest of the programming languages, the condition is not surrounded by paranthesis.
```
Comment (SSwiniarski) [inline content/rust/concepts/conditionals/conditionals.md:15]:
```suggestion
**Conditional** statements in Rust allow the control of flow in program based on certain conditions. Rust provides support for Conditional statements that include `if`, `else if` and `else` statements. Unlike many programming languages, the condition is not surrounded by parenthesis.
```

File: content/rust/concepts/conditionals/conditionals.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+if condition {
+  // Code to be executed if condition is true
+} else if condition1 {
+  // Code to be executed if condition1 is true
+} else if condition2 {
+  // Code to be executed if condition2 is true
+}
+```
+
+Example of `else if` statement:
```
Comment (SSwiniarski) [inline content/rust/concepts/conditionals/conditionals.md:55]: Move this to after the "Example" header.

File: content/rust/concepts/conditionals/conditionals.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+let number = 15;
+
+if number > 40 {
+  println!("The number is greater than 40.");
+}
+else if number > 20 {
+  println!("The number is greater than 20.");
+}
+else {
+  println!("This line will be executed.");
```
Comment (SSwiniarski) [inline content/rust/concepts/conditionals/conditionals.md:101]:
```suggestion
  println!("The number is 20 or less.");
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Captainnik53 I did a first review and left you a few comments.

**Comment by Captainnik53** [comment]
> @Captainnik53 I did a first review and left you a few comments.

Have resolved all issues

File: content/rust/concepts/conditionals/conditionals.md (hunk 3 above)
Comment (Captainnik53) [reply content/rust/concepts/conditionals/conditionals.md]: Done

**Review by SSwiniarski — APPROVED** [review]
@Captainnik53 i DID ONE MINOR EDIT AND IT LOOKS LIKE WE'RE READY FOR A SECOND REVIEW.

File: content/rust/concepts/conditionals/conditionals.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+Description: 'Conditional statements are the decision-making statements that evaluate an expression to provide control flow in a program.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Conditionals'
+  - 'Control Flow'
+CatalogContent:
+  - 'paths/computer-science'
+  - 'learn/rust-for-programmers'
```
Comment (caupolicandiaz) [inline content/rust/concepts/conditionals/conditionals.md:12]:
```suggestion
  - 'rust-for-programmers'
  - 'paths/computer-science'
```

File: content/rust/concepts/conditionals/conditionals.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+---
+
+**Conditional** statements in Rust allow the control of flow in program based on certain conditions. Rust provides support for Conditional statements that include `if`, `else if` and `else` statements. Unlike many programming languages, the condition is not surrounded by parenthesis.
```
Comment (caupolicandiaz) [inline content/rust/concepts/conditionals/conditionals.md:15]:
```suggestion
**Conditional** statements in Rust allow the control of flow in program based on certain conditions. Rust provides support for conditional statements that include `if`, `else if` and `else` statements. Unlike many programming languages, the condition is not enclosed by parentheses.
```

File: content/rust/concepts/conditionals/conditionals.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+} else if condition1 {
+  // Code to be executed if condition1 is true
+} else if condition2 {
+  // Code to be executed if condition2 is true
+}
+```
+
+### Example
+
+Example of `else if` statement:
```
Comment (caupolicandiaz) [inline content/rust/concepts/conditionals/conditionals.md:57]:
```suggestion
Example of an `else if` statement:
```

File: content/rust/concepts/conditionals/conditionals.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+} else if condition1 {
+  # Code to be executed if condition1 is true
+} else {
+  # Code to be executed if all above conditions are false
+}
+```
+
+### Example
+
+Example of `else` statement:
```
Comment (caupolicandiaz) [inline content/rust/concepts/conditionals/conditionals.md:89]:
```suggestion
Example of an `else` statement:
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Captainnik53 I've added a few additional comments. Please update the entry, and tag me when you've finished making the changes. Thanks

**Comment by Captainnik53** [comment]
> @Captainnik53 I've added a few additional comments. Please update the entry, and tag me when you've finished making the changes. Thanks

@caupolicandiaz Have done changes as per your comments.

**Review by caupolicandiaz — APPROVED** [review]
@Captainnik53 @SSwiniarski LGTM

**Pull request merged**